js/ResourcesHelpers.js

```diff
@@ -0,0 +1,43 @@
+import Handlebars from 'handlebars';
+import Adapt from 'core/js/adapt';
+
+class ResourcesHelpers {
```

Comment:
This should be left as an `Object` as it never needs to be instantiated, it's just acting as "hash" / a named list for a loop.